src/main/java/uk/gov/hmcts/dm/service/batch/AuditedStoredDocumentBatchOperationsService.java

```diff
@@ -4,12 +4,14 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
 import uk.gov.hmcts.dm.domain.AuditActions;
 import uk.gov.hmcts.dm.domain.StoredDocument;
 import uk.gov.hmcts.dm.service.AuditEntryService;
 import uk.gov.hmcts.dm.service.StoredDocumentService;
 
 @Service
+@Transactional
```

Comment:
I am not sure. If this is required. But should not impact other bits. As this service has method/logic only related to Batch.